paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/cdc/MultiplexCdcRecord.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.flink.sink.cdc;
+
+import org.apache.paimon.types.RowKind;
+
+import java.util.Map;
+
+/**
+ * A data change message from the CDC source. Compared to {@link CdcRecord}, MultiplexCdcRecord
+ * contains database and table information.
+ */
+public class MultiplexCdcRecord extends CdcRecord {
```

Comment:
Can we use just create a field `CdcRecord ` instead of extending?
Extending will make the caller think it can be used as a CdcRecord.

Reply:
Yes a field is better than extending here.